### ISSUE TITLE
Analysis config and report updates

### ISSUE THREAD

**Issue by meganrm** [mesoscope/cellpack#132]
## Use Case
We want to run all our available packing modes on a test recipe and have a report generated for PRs to be able to see any changes in the packing results introduced by the code changes. 

## Solution
`pack -r RECIPE -config CONFIG -p [LIST OF PACKING MODES]`
`analyze -r RESULTES_FOLDER -config CONFIG` 
analyze will loop through all subdirectories in the results folder and generate a report for each packing mode, which all get included in one PR comment. 

```JSON
{
    "version": "1.0.0",
    "format_version": "1.0",
    "input_path": "cellpack/tests/outputs/test_spheres/jitter",
    "output_path": "cellpack/tests/outputs/test_spheres",
    "ingredient_key" : "ext_A",
    "create_report": {
        "run_distance_analysis": true,
        "path_to_test_recipe": "cellpack/tests/recipes/test_spheres.json", <= remove
        "path_to_results": "cellpack/tests/outputs/test_spheres/jitter", <= remove this, it's redundant with `input_path`
        "results_output_path": "https://cellpack-results.s3.us-west-2.amazonaws.com/jitter"
    }
}
```
-QUESTION: can we style the images so they are side by side to reduce the overall size of the comment?

## Alternatives
*Please describe any alternatives you've considered, even if you've dismissed them*

**Comment by meganrm** [comment]
next step after #78 completes  #35

**Comment by mogres** [comment]
Update the `create_report` function to accept the input dictionary as keyword arguments.

**Comment by mogres** [comment]
- [ ] Check if PR exists before running comment workflow

**Comment by mogres** [comment]
> * [ ]  Check if PR exists before running comment workflow

This is addressed by #150
